config/locales/candidate_interface/restructured_work_history.yml

```diff
@@ -11,7 +11,35 @@ en:
       explanation:
         label: Tell us why you’ve been out of the workplace
         hint_text: For example, ‘I have been unemployed’, ‘I have been volunteering’ or ‘I have been caring for a child’
-
+      role:
+        label: Role
+        review_label: Role
+        change_action: role
+        hint_text: If you had more than one role with this employer, enter your last role.
+      employer:
+        label: Name of employer
+        hint_text: If you worked for yourself, enter ‘self-employed’.
+        review_label: Employer
+        change_action: employer
+      commitment:
+        label: Was this job full time or part time?
+        full_time:
+          label: Full time (usually at least 35 hours a week)
+        part_time:
+          label: Part time
+      start_date:
+        label: When did you start this job?
+        hint_text: For example, 5 2018. If you cannot remember the exact month or year, enter an estimate.
+      start_date_unknown_checkbox: I'm not sure the exact month or year I started
+      end_date:
+        label: When did you leave this job?
+        hint_text: For example, 5 2018. If you cannot remember the exact month or year, enter an estimate.
+      end_date_unknown_checkbox: I'm not sure the exact month or year I left
```

Comment:
`I’m not sure the exact month or year I left`

Reply:
👍